src/lib.rs

```diff
@@ -0,0 +1,141 @@
+extern crate diesel;
+extern crate byteorder;
+extern crate geo;
+
+use byteorder::{ByteOrder, LittleEndian, BigEndian};
+use std::error::Error;
+use std::io::prelude::*;
+use diesel::pg::{Pg, PgTypeMetadata};
+use diesel::query_builder::QueryId;
+use diesel::row;
+use diesel::types::{FromSql, ToSql, IsNull, HasSqlType, FromSqlRow};
+use std::fmt;
+use geo::*;
+
+
+pub struct Geography {
+    pub big_endian: bool,
+    pub srid: Option<u32>,
+    pub geom: geo::Geometry<f64>,
+}
+
+#[derive(Debug)]
+struct NotImplemented {}
+impl fmt::Display for NotImplemented {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "Not implemented Error")
+    }
+}
+
+impl Error for NotImplemented {
+    fn description(&self) -> &str {
+        "Not implemented"
+    }
+}
+
+impl Geography {
+    pub fn from(bytes: &[u8]) -> Result<Self, Box<Error + Send + Sync>> {
```

Comment:
Thanks! For now that function disapeared, but I expect that it will show up at some point